docs/operatorhub.md

```diff
@@ -0,0 +1,59 @@
+# Publishing a new version on OperatorHub
+
+## Overview
+The process to publish a version on [OperatorHub](https://operatorhub.io/) is as following:
+
+- Prepare the files in `knative/eventing` repository
+- Test locally
+- Send a PR to https://github.com/operator-framework/
+- Once that passes the Prow tests and gets approved by the OperatorHub people, send a PR to `knative/eventing` so that we also have the metadata handy in this repository
+
```

Comment:
<!--mdfmt-bot-->
Format markdown:
```suggestion
- Once that passes the Prow tests and gets approved by the OperatorHub people,
  send a PR to `knative/eventing` so that we also have the metadata handy in
  this repository
```